docs/4___co-simulation.adoc

```diff
@@ -1,93 +1,48 @@
 == FMI for Co-Simulation [[fmi-for-co-simulation]]
 
-This chapter defines the Functional Mock-up Interface (FMI) for the coupling of two or more simulation models in a co-simulation environment (FMI for Co-Simulation).
-It is designed both for coupling with subsystem models, which have been exported by their simulator together with its solver as runnable code, and for coupling of simulation tools on a single machine or as part of a distributed co-simulation.
+FMI for Co-Simulation provides a standardized interface for execution of simulation models or tools in a co-simulation environment.
+In contrast to FMI for Model Exchange, a Co-Simulation FMU implements not only the model algorithm, but also the required solution method.
+The data exchange between FMUs is restricted to discrete communication points latexmath:[\mathbf{t}_i].
```

Comment:
"The data exchange between FMUs is restricted to discrete communication points": this is only true for the "direct" data exchange, but via the importer in intermediateUpdateMode, data can be exchanged somehow.

Reply:
Indeed. I tried to improve the text there, please check and close this comment if you are happy or propose further improvements otherwise.

Reply:
change of tactics: I am closing, if you are unhappy, please reopen.